spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/RDDFunctions.scala

```diff
@@ -152,10 +155,12 @@ class RDDFunctions[T](rdd: RDD[T]) extends WritableToCassandra[T] with Serializa
    * of the data specified by that row.
    * The calling RDD must have rows that can be converted into the partition key of the given Cassandra Table.
    */
-  def keyByCassandraReplica(keyspaceName: String, tableName: String)
+  def keyByCassandraReplica(keyspaceName: String,
```

Comment:
This matches the style of the other functions in the file I think. Would you like me to switch the whole file to
```scala
def foo(
  arg1
  arg2
):type = {

}
```